README.md

```diff
@@ -116,7 +116,15 @@ async function keyProtectSdkExample() {
   const getKeyParams = Object.assign({}, envConfigs);
   getKeyParams.id = keyId;
   response = await keyProtectClient.getKey(getKeyParams);
-  console.log('Get key response status: ' + response.status);
+  console.log('Get key result: ');
+  console.log(response.result.resources[0]);
```

Comment:
Should we be checking for the response being valid before accessing it?

Reply:
It should be fine - if the response is not good it will throw error when accessing the data in it